conference.js

```diff
@@ -2219,7 +2219,7 @@ export default {
 
         // call hangup
         APP.UI.addListener(UIEvents.HANGUP, () => {
-            this.hangup(true);
```

Comment:
the hangup method checking the config property "feedbackPercentage"

if this property not exists is would be set to 100%

If you set this property feedbackPercentage = 0 in the config.js, i think the feedback will not triggert. 

'feedbackPercentage' is listed as extra property

https://github.com/jameda-telemedicine/jitsi-meet/blob/f0e9d17fbba2c6dc273d2542496b98e2040a5ef7/react/features/base/config/configWhitelist.js

Reply:
Please have a look in Line 57

https://github.com/jameda-telemedicine/jitsi-meet/blob/f0e9d17fbba2c6dc273d2542496b98e2040a5ef7/react/features/feedback/actions.js

Reply:
That's true, I will revert my changes.